teams/tikv/team.json

```diff
@@ -53,7 +53,7 @@
         "Fullstop000",
         "Jibbow",
         "Xuanwo",
-        "accelsao",
+        "jayzhan211",
```

Comment:
What if other users change name and don't inform us? I worry that the file is critical to permission and if others reclaim the user name then can do harm to all the projects.